index.js

```diff
@@ -59,6 +64,11 @@ async function run() {
     const version = core.getInput('version');
     const arch = core.getInput('arch');
     const gitUrl = core.getInput('repoUrl');
+    const buildSource = core.getBooleanInput('forceBuildFromSource');
+    const userBuildArgs = core.getInput('gstreamerOptions')
+      .split(/[\r\n]/)
+      .map(input => input.trim())
+      .filter(input => input !== '');
```

Comment:
Use `core.getMultilineInput()`.
```suggestion
    const userBuildArgs = core.getMultilineInput('gstreamerOptions');
```